loadtest/config.go

```diff
@@ -48,6 +48,10 @@ type DeploymentConfiguration struct {
 	ClusterName      string // Name of the cluster.
 	AppInstanceCount int    // Number of application instances.
 	SSHPublicKey     string // Path to the SSH public key.
+	InstanceCount    int    // Number of DB instances.
+	InstanceEngine   string // Type of the DB instance - postgres or mysql.
```

Comment:
we should validate the values passed (aurora-postgres, mysql, etc. if user sets it to some random stuff, errors will appear)

Reply:
done